modules/kernel/src/main/scala/tofu/Fire.scala

```diff
@@ -2,15 +2,23 @@ package tofu
 
 import scala.util.Either
 import tofu.internal.{EffectComp, Effect3Comp}
-import tofu.internal.carriers.FibersCarrier
+import tofu.internal.carriers.FibersCarrier2
+import cats.MonadError
+import tofu.internal.carriers.FibersCarrier3
+import scala.annotation.unused
 
 trait Fire[F[_]] {
   def fireAndForget[A](fa: F[A]): F[Unit]
 }
 
 object Fire extends EffectComp[Fire] {
-  final implicit def byCarrier[F[_], Ex[_], Fib[_]](implicit
-      carrier: FibersCarrier.Aux[F, Ex, Fib]
+  final implicit def byCarrier2[F[_], Ex[_], Fib[_]](implicit
```

Comment:
Could you use postfix `CE2` & `CE3`? I think it makes the intention clearer for a reader.
Like so `byCarrierCE2`

Reply:
okaaay